.github/workflows/scripts/codepath-notification

```diff
@@ -13,3 +13,7 @@
 # The aim is to find a minimal set of regex patterns that matches any file in these paths
 
 rubicon: header-bidding@magnite.com
+pubmatic: header-bidding@pubmatic.com
+openx: prebid@openx.com
+/adapters/ix|imp_ix|ix.json|ix.yaml: pdu-supply-prebid@indexexchange.com
```

Comment:
Should the first forward slash be there? Should this be `adapters/ix|imp_ix|ix.json|ix.yaml` instead?

My manual regex testing shows that the current pattern is transformed into `/\/adapters\/ix|imp_ix|ix.json|ix.yaml/` by the `new RegExp(regex)` call in `.github/workflows/scripts/send-notification-on-change.js` and it looks like that doesn't match files in `adapters/ix` when I manually run `regex.test`.

Reply:
Fixed,.